src/components/Bar.svelte

```diff
@@ -4,7 +4,7 @@
 </button>
 {{/if}}
 
-<h1 class='{{titleClass}}'>
+<h1 data-lang='{{lang}}' class='{{titleClass}}'>
```

Comment:
In fact I added this attribute just to use the lang data property in the component template to get the correct rerendering behaviour when the lang change. I don't have opinions about that, just I know data-lang is also accepted as attribute. So let's do it 👍